### ISSUE TITLE
Revamp important documents section

### ISSUE THREAD

**Issue by sachin2dehury** [dscnitrourkela/project-avocado#191]
instead of urls, load thumbnails .

**Comment by DesignrKnight** [comment]
Hey @sachin2dehury 
Why close this? Is this resolved in an existing PR or is feature cancelled to be implemented?

**Comment by DesignrKnight** [comment]
Hey @abhibhaw 
This looks like a good feature. Discuss with one is the new developers and guide them on this.

If someone is willing to work, open the issue and work

**Comment by DesignrKnight** [comment]
CC: @SriramPatibanda

**Comment by abhibhaw** [comment]
Yep this should be implemented, @sachin2dehury it would be great if you can add a bit more details about the desired outcome. ;)

**Comment by Chinmay-KB** [comment]
When we share a doc on whatsapp, the thumbnail is the first page of that doc, something similar here maybe. In place of list of urls and titles, a list of images and titles
